src/sssom/rdf_util.py

```diff
@@ -42,8 +43,14 @@ def rewire_graph(
                 curr_tgt = rewire_map[src]
                 logging.info(f"Ambiguous: {src} -> {tgt} vs {curr_tgt}")
                 if precedence:
-                    curr_pfx, _ = converter.parse_curie(curr_tgt)
-                    tgt_pfx, _ = converter.parse_curie(tgt)
+                    curr_reference = converter.parse_curie(curr_tgt)
+                    if curr_reference is None:
+                        raise ValueError
```

Comment:
Would it make sense to provide a somewhat meaningful message here?

Reply:
Good point. Better error messages are now generated by a newer version of `curies`, so I bumped the minimum version to take care of that in d355756